### PR TITLE
Create my About Me page

### PR DESCRIPTION
**Pull request by bernars-usa**
I copied this repo from Sheri's template repo. I'm just filling out the default information. I may add more later in a future pull request.

Let me know if you have any questions!

@bestes-usds for your review. 👀 

🐃 🎏 😳

### CODE REVIEW

File: about-me.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
+![Picture of me](https://avatars1.githubusercontent.com/u/40181132?s=460&v=4)
 
-Hi, I'm ____________. I'm a ________ at *__________*.
+Hi, I'm Charlene B. Trivedi. I'm a content strategist at *OPM*.
 
-- **Past Companies and Organizations:**
-- **Hobbies:**
-- **Favorite Website:** []()
+- **Past Companies and Organizations:** 
+- **Hobbies:** Reading, writing, singing in the car
```
Comment (bestes-usds) [inline about-me.md:8]: Turn the radio up!

File: about-me.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 # About Me
 
-![Picture of me]()
+![Picture of me](https://avatars1.githubusercontent.com/u/40181132?s=460&v=4)
 
-Hi, I'm ____________. I'm a ________ at *__________*.
+Hi, I'm Charlene B. Trivedi. I'm a content strategist at *OPM*.
```
Comment (bestes-usds) [inline about-me.md]: Charlene? Is that an error?

**Comment by bestes-usds** [comment]
Just had one question for your review

**Comment by bestes-usds** [comment]
Just one thing for your review

File: about-me.md (hunk 2 above)
Comment (bernars-usa) [reply about-me.md]: Ah, yeah. Charlene is my legal name. Updated for clarity, thanks for catching that!

**Pull request merged**